go/throttle/throttler.go

```diff
@@ -250,6 +256,16 @@ func (throttler *Throttler) aggregateMySQLMetrics() error {
 		ignoreHostsCount := throttler.mysqlInventory.IgnoreHostsCount[clusterName]
 		aggregatedMetric := aggregateMySQLProbes(probes, throttler.mysqlInventory.InstanceKeyMetrics, ignoreHostsCount)
 		go throttler.aggregatedMetrics.Set(metricName, aggregatedMetric, cache.DefaultExpiration)
+		if throttler.memcacheClient != nil {
+			go func() {
+				value, err := aggregatedMetric.Get()
+				if err != nil {
+					throttler.memcacheClient.Set(&memcache.Item{Key: metricName, Value: []byte("")})
```

Comment:
So, would you have the value as `1497276176059930561:0.52`, where `1497276176059930561` is a unix epoch (nanoseconds) and `0.52` is the value?